notebooks/fireredtts2/fireredtts2.ipynb

```diff
@@ -0,0 +1,1312 @@
+{
```

Comment:
<p>Could you, please, add description about converted models ?  What is the purpose of this model within the overall model ? Are there any specific when converting to IR?</p><p>I mean TEXT/AUDIO EMBEDDINGS, AUDIO_DECODER , AUDIO_UPSAMPLER, AUDIO_ENCODER, DECODER_MODEL, BACKBONE_MODEL.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3120/discussion/#comment-2571762292'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>